lib/main_common.pm

```diff
@@ -649,6 +649,7 @@ sub load_wicked_tests {
 sub load_nfv_tests {
     boot_hdd_image();
     loadtest "nfv/openvswitch_dpdk";
+    loadtest "nfv/vsperf_installation";
```

Comment:
If I correctly understand there is no point running vsperf_installation module in case openvswitch_dpdk will fail, right ?
In this case you probably need to add this section to openvswitch_dpdk :
```
sub test_flags {
    return {fatal => 1};
}
```
It will instruct openQA to stop execution after module failure

Reply:
done